golang/src/com/google/idea/blaze/golang/run/BlazeGoRunConfigurationRunner.java

```diff
@@ -409,6 +409,8 @@ private static File findExecutable(Label target, List<File> outputs) {
   private static final Pattern TEST_SRCDIR = Pattern.compile("TEST_SRCDIR=([^ ]+)");
   // Matches RUNFILES_<NAME>=<value>
   private static final Pattern RUNFILES_VAR = Pattern.compile("RUNFILES_([A-Z_]+)=([^ ]+)");
+  // Matches TEST_TARGET=//<package_path>:<target>
+  private static final Pattern TEST_TARGET = Pattern.compile("TEST_TARGET=//([^:]*):([^\\s]+)");
```

Comment:
We have bzlmod in some form in our company's repo which I have also tested this on, however I'm not familiar with what it does and what implications it can bring. So, no, and I don't really know how to test this extensively.

Reply:
So the main difference between bzlmod and non-bzlmod projects is the introduction of [canonical repository names](https://bazel.build/external/overview#canonical-repo-name). Also some rulesets behave differently depending on whether bzlmod is enabled or not (however user-visible differences are usually a bug).

I was asking because whenever there's a target pattern with a single `@` or without any `@`s it is possible that  under bzlmod it got turned into a `@@` (we - JetBrains - have run into such problems in out [Hirschgarten](https://github.com/JetBrains/hirschgarten/) repo).

But if it works with bzlmod it's much more probable that it will work without it than the other way round :)

So I'll merge it if it works for you!